tsconfig.eslint.json

```diff
@@ -3,5 +3,5 @@
   "compilerOptions": {
     "noEmit": true
   },
-  "include": ["src", "test", "tools", "bin"]
+  "include": ["src", "packages/*/tests"]
```

Comment:
ça pourrait pas être juste ça?

"include": [ "packages/*"]

Reply:
Maybe it requires revisiting some `tsconfig.json` file include/exclude scope, but just having `package/*` break eslint completely